From ab53e1b380854edaef1d3bc512b2f5d405d07573 Mon Sep 17 00:00:00 2001
From: mark gross <mark.gross@intel.com>
Date: Thu, 15 May 2014 13:08:33 -0700
Subject: [PATCH] From: Dmitriy Berchanskiy <dberchan@intel.com> Date: Wed, 18
 Dec 2013 09:21:32 -0800 Subject: [PATCH] Realtek: Added RTL8723 support for
 wifi-bt combo ic

BZ: 151767

Add sdio platform regulator and ACPI support for RTL8723

Change-Id: Ifd4d1e796adc4e109268fc989951a8f1056c4c6d
Category: device enablement
Domain: CWS.WIFI-Common
Origin: internal
Upstream-Candidate: no, proprietary
Signed-off-by: Dmitriy Berchanskiy <dberchan@intel.com>
---
 arch/x86/platform/intel-mid/device_libs/platform_sdio_regulator.c | 1 +
 drivers/acpi/acpi_platform.c                                      | 2 ++
 2 files changed, 3 insertions(+)

diff --git a/arch/x86/platform/intel-mid/device_libs/platform_sdio_regulator.c b/arch/x86/platform/intel-mid/device_libs/platform_sdio_regulator.c
index 01953a6..e2cf8e9 100644
--- a/arch/x86/platform/intel-mid/device_libs/platform_sdio_regulator.c
+++ b/arch/x86/platform/intel-mid/device_libs/platform_sdio_regulator.c
@@ -32,6 +32,7 @@ static struct acpi_ids intel_sdio_ids[] = {
 
 static struct acpi_ids intel_brc_ids[] = {
 	{"BCM4321", NULL}, /* BYT SDIO */
+	{"RTL8723" , NULL}, /* BCR SDIO */
 	{ },
 };
 
diff --git a/drivers/acpi/acpi_platform.c b/drivers/acpi/acpi_platform.c
index 9967493..67dac8c 100644
--- a/drivers/acpi/acpi_platform.c
+++ b/drivers/acpi/acpi_platform.c
@@ -36,6 +36,8 @@ static const struct acpi_device_id acpi_platform_device_ids[] = {
 	{ "BCM2E1A" },
 	{ "BCM2E39" },
 	{ "BCM2E3D" },
+	{ "RTL8723" },
+	{ "OBDA8723" },
 
 	/* Intel Smart Sound Technology */
 	{ "INT33C8" },
-- 
1.9.3

